src/components/Navigation/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import * as ROUTES from '../../constants/routes';
+
+import SignOutButton from '../SignOut';
+import { SignInForm } from '../SignIn';
+
+const Navigation = ({ authUser} ) => (
+        <div>{authUser ? <NavigationAuth /> : <br />}</div>
+);
+
+const NavigationAuth = () => (
+    <div>
+        <h2>Navigation</h2>
+        <ul>
+            <li>
+                <Link to={ROUTES.ACCOUNT}>Account</Link>
```

Comment:
Using constants like this seems like a good idea. I think I'll steal this idea for the future

Reply:
It seems to be a really easy way to keep track of the different routes.